### PR TITLE
docs: Add new settings

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
## Proposed changes

Describe recently added settings

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: docs/en/advanced-concepts/settings.md [hunk 1]
```diff
@@ -91,7 +92,9 @@ Settings are implemented via the following API endpoints:
...
-|`snapshotTimeout` | Changes the accessibility snapshots resolution timeout. _Snapshots_ are mainly used for page source generation, XML lookup and attributes retrieval. It might be necessary to increase this value if the actual page source is very large and contains hundreds of UI elements. Defaults to 15 seconds. | e.g. `10`, `100` (seconds) |1.15.0+|
+|`customSnapshotTimeout` (`snapshotTimeout` before 1.19.1) | Set how much time is allowed to resolve a single accessibility snapshot with custom attributes. _Snapshots_ are mainly used for page source generation, XML lookup and custom attributes retrieval (these are visibility and accessibility ones). It might be necessary to increase this value if the actual page source is very large and contains hundreds of UI elements. Defaults to 15 seconds. Since Appium 1.19.1 if this timeout expires and no custom snapshot could be made then WDA tries to calculate the missing attributes using its own algorithms, so setting this value to zero might speed up, for example, page source retrieval, but for the cost of some element attribute preciseness. | e.g. `10`, `100` (seconds) |1.15.0+|
```
Comment (KazuCocoa) [inline docs/en/advanced-concepts/settings.md:95]: 📓 
We should guide users to switch this capability with the new name in the next release note

File: docs/en/advanced-concepts/settings.md [hunk 2]
```diff
@@ -91,7 +92,9 @@ Settings are implemented via the following API endpoints:
...
+|`waitForIdleTimeout`|The amount of time in float seconds to wait until the application under test is idling. XCTest requires the app's main thread to be idling in order to execute any actions on it, so each action, like single click, might take a lot of time in case your app is constantly hogging the main thread. The default value is `10`. Setting it to zero disables idling checks completely (not recommended).|e.g. `2.5`|1.19.1|
+|`animationCoolOffTimeout`|The amount of time in float seconds to wait until the application under test does not have any active animations. This check is usually applied after each action, like  `click` one and is started in parallel with the idling check (see `waitForIdleTimeout` setting). The default value is `2`. Setting it to zero disables animation checks completely.|e.g. `1.5`|1.19.1|
```
Comment (KazuCocoa) [inline docs/en/advanced-concepts/settings.md:97]: It would be nice to add `0` (zero) case in the `e.g.` section, too

**Review by KazuCocoa — APPROVED** [review]

File: docs/en/advanced-concepts/settings.md (hunk 1 above)
Comment (mykola-mokhnach) [reply docs/en/advanced-concepts/settings.md:95]: the old cap name is still there, I did not completely remove it

File: docs/en/advanced-concepts/settings.md (hunk 2 above)
Comment (mykola-mokhnach) [reply docs/en/advanced-concepts/settings.md:97]: added

**Review by KazuCocoa — APPROVED** [review]

**Pull request merged**